nights/2020-02-19-README.md

```diff
@@ -1,39 +1,13 @@
 # 2020-02-19
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Mara DeRung
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2020-02-19
 
 ## Unusual images?
 
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
 
-+ `crazy-object-001R.fit` -- not sure what this is an image of
-+ `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
-+ `full-moon-005B.fit` -- nice satellite or plane track in here.
-
-Delete this list if there are no unusual images.
-
-## Missing information?
```

Comment:
Can you please put this section back? You might need to copy/paste from here: https://raw.githubusercontent.com/feder-observatory/processed_images/master/nights/2011-02-09-README.md